examples/pytorch/language-modeling/run_mlm.py

```diff
@@ -476,6 +476,22 @@ def group_texts(examples):
         if data_args.max_eval_samples is not None:
             eval_dataset = eval_dataset.select(range(data_args.max_eval_samples))
 
+        def preprocess_logits_for_metrics(logits, labels):
+            return logits.argmax(dim=-1)
+
+        metric = load_metric("accuracy")
+
+        def compute_metrics(eval_preds):
+            preds, labels = eval_preds
+            # preds have the same shape as the labels, after the argmax(-1) has been calculated
+            # by preprocess_logits_for_metrics
```

Comment:
Same as above.

Reply:
Here I added the same comment but I'm not shifting the labels. It's not needed, right?

Reply:
Ah yes, not here sorry.